_data/governments.yml

```diff
@@ -728,6 +728,7 @@ U.S. Federal:
   - gsa-oes
   - hhs
   - HHS-AHRQ
+  - HHS-NIH-NHGRI
```

Comment:
https://github.com/HHS-NIH-NHGRI doesn't exist. Is there a typo or in the org name or has it been deleted or renamed?